.github/workflows/reusable-git-object-name.yml

```diff
@@ -19,6 +19,5 @@ jobs:
         id: set_outputs
         shell: bash -l {0}
         run: |
-          git fetch origin +refs/tags/*:refs/tags/*
```

Comment:
`actions/checkout@v3` fetches all the tags so this is no longer needed.

Reply:
https://github.com/actions/checkout/blob/1f9a0c22da41e6ebfa534300ef656657ea2c6707/README.md#fetch-all-history-for-all-tags-and-branches